### PR TITLE
Update Contract Names & Fix Manager Role Address Bug

### PR DESCRIPTION
**Pull request by kamescg**

### CODE REVIEW

File: hardhat.network.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { HardhatUserConfig } from 'hardhat/config';
 const mnemonic = process.env.HDWALLET_MNEMONIC;
+const infuraApiKey = process.env.INFURA_API_KEY;
```
Comment (asselstine) [inline hardhat.network.ts:3]: why was this re-introduced? not needed

File: hardhat.network.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+const avalanche = process.env.AVALANCHE_ENABLED;
```
Comment (asselstine) [inline hardhat.network.ts:4]: why is this necessary?

File: hardhat.network.ts [hunk 3]
```diff
@@ -29,15 +24,25 @@ const networks: HardhatUserConfig['networks'] = {
...
+  polygon: {
+    chainId: 137,
+    url: process.env.POLYGON_RPC_URL,
+    accounts: {
+      mnemonic,
+    },
+  },
+};
+
+if (!!avalanche) {
```
Comment (asselstine) [inline hardhat.network.ts:36]: why is this conditional?  this was just fixed

File: deploy/v1.1.0/avalanche.ts [hunk 4]
```diff
@@ -31,6 +31,15 @@ export default async function deployToAvalanche(hardhat: HardhatRuntimeEnvironme
...
   } = await getNamedAccounts()
 
+  console.log(
+    deployer,
+    executiveTeam,
+    aUSDC,
+    defenderRelayer,
+    aaveIncentivesController,
+    aaveLendingPoolAddressesProviderRegistry
+  )
```
Comment (asselstine) [inline deploy/v1.1.0/avalanche.ts:41]: Use conventions elsewhere in the code.

See here: https://github.com/pooltogether/v4-mainnet/blob/699abdfb5c0f07a6b5dddc3a4de022f92fc08d68/deploy/v1.0.1/polygon.ts#L36

**Review by asselstine — CHANGES_REQUESTED** [review]

File: hardhat.network.ts (hunk 3 above)
Comment (kamescg) [reply hardhat.network.ts:36]: It's a conditional so Hardhat doesn't error if the `.envrc` is not setup with a provider URL.

File: hardhat.network.ts (hunk 1 above)
Comment (kamescg) [reply hardhat.network.ts:3]: I'm not sure why it was was re-introduced? It was just present in my files.

Removed.

File: deploy/v1.1.0/avalanche.ts (hunk 4 above)
Comment (kamescg) [reply deploy/v1.1.0/avalanche.ts:41]: ohh this should be ✂️  was double checking an error with an empty param.

**Review by asselstine — APPROVED** [review]
